Dockerfiles/agent/Dockerfile

```diff
@@ -21,32 +24,39 @@ RUN apt-get update \
 #   - static libraries                   # FIXME: move upstream
 #   - jmxfetch on nojmx build
 RUN dpkg -x /datadog-agent*_amd64.deb . \
- && echo "Cleanup and trimming" \
  && rm -rf usr etc/init lib \
     opt/datadog-agent/sources \
     opt/datadog-agent/embedded/share/doc \
     opt/datadog-agent/embedded/share/man \
  && find opt/datadog-agent/ -iname "*.a" -delete \
  && if [ -z "$WITH_JMX" ]; then rm -rf opt/datadog-agent/bin/agent/dist/jmx; fi \
- && echo "Linking embedded ssl certificates" \
- && ln -s /opt/datadog-agent/embedded/ssl etc/ssl
+ && ln -s /opt/datadog-agent/embedded/ssl etc/ssl \
+ && mkdir conf.d checks.d
 
 # Configuration:
-#   - empty config file to remove logline
+#   - copy default config files
 #   - enable docker check by default
 COPY datadog*.yaml etc/datadog-agent/
 RUN mv etc/datadog-agent/conf.d/docker.d/conf.yaml.example etc/datadog-agent/conf.d/docker.d/conf.yaml.default
 
-COPY entrypoint.sh probe.sh /output/
-
 
+####################################
+# Actual docker image construction #
+####################################
 
 FROM debian:stretch-slim
 LABEL maintainer "Datadog <package@datadoghq.com>"
 ARG WITH_JMX
 ENV DOCKER_DD_AGENT=yes \
     PATH=/opt/datadog-agent/bin/agent/:/opt/datadog-agent/embedded/bin/:$PATH \
-    CURL_CA_BUNDLE=/opt/datadog-agent/embedded/ssl/certs/cacert.pem
+    CURL_CA_BUNDLE=/opt/datadog-agent/embedded/ssl/certs/cacert.pem \
+    TERM=xterm \
```

Comment:
I think we can remove this it's already the default in `debian:stretch-slim`

Reply:
It actually didn't work for me, `ps aux` only used 80 colums by default. Dunno if the issue is with my host system :/